extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/util/AccessControlException.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.plugin.spark.authz.util
+
+/**
+ * This exception is thrown by the Kyuubi AuthZ module to indicate
+ * that a requested access is denied.
+ */
+class AccessControlException(msg: String, cause: Option[Throwable] = None)
+  extends Exception(msg, cause.orNull) {}
```

Comment:
Usually we are not encouraged to use checked exception, could you please change it to `extends RuntimeException` instead?
Ref: https://phauer.com/2015/checked-exceptions-are-evil/

Reply:
thanks